scripts/qa_crowbarsetup.sh

```diff
@@ -525,7 +525,7 @@ function addsp3testupdates()
     add_mount "SLES11-SP3-Updates-test" \
         $distsuseip':/dist/ibs/SUSE:/Maintenance:/Test:/SLE-SERVER:/11-SP3:/x86_64/update/' \
         "$tftpboot_repos_dir/SLES11-SP3-Updates-test/" "sp3tup"
-    [ -n "$hacloud" ] && add_mount "SLE11-HAE-SP3-Updates-test" \
```

Comment:
Just for the record: this changes the semantics.

I that `['', '1']` are documented as allowed values. We had a feature using `hacloud=2`. Thats the reason why it still says `"-n $hacloud"`.
So this change will break setups where people still use `hacloud=$nonemptystring`

I am not arguing against this change, we should just be aware of this as there might be issues or questions popping up.